camelot/view/controls/delegates/floatdelegate.py

```diff
@@ -65,16 +67,21 @@ def get_standard_item(cls, locale, model_context):
         item.setData(model_context.field_attributes.get('focus_policy'), FocusPolicyRole)
         item.setData(model_context.field_attributes.get('suffix'), SuffixRole)
         item.setData(model_context.field_attributes.get('prefix'), PrefixRole)
-        item.setData(model_context.field_attributes.get('single_step'), SingleStepRole)
+        single_step = model_context.field_attributes.get('single_step')
+        if isinstance(single_step, Decimal):
+            single_step = float(single_step) # FIXME: use bound decimal?
+        item.setData(single_step, SingleStepRole)
         precision = model_context.field_attributes.get('precision', 2)
         item.setData(precision, PrecisionRole)
-        item.setData(minimum, MinimumRole)
-        item.setData(maximum, MaximumRole)
+        item.setData(float(minimum), MinimumRole) # FIXME: use bound decimal?
```

Comment:
idd, we kunnen er in onze use case van uit gaan dat float nt bestaat, en alles decimal is